crates/extensions/c8y_mapper_ext/src/actor.rs

```diff
@@ -108,19 +109,25 @@ impl C8yMapperActor {
     }
 
     async fn process_mqtt_message(&mut self, message: MqttMessage) -> Result<(), RuntimeError> {
-        // to which entity is the message related
         if let Some(entity_id) = entity_store::entity_mqtt_id(&message.topic) {
-            // if message is registration message:
             if is_entity_register_message(&message) {
-                // update entity store with registration message
-                // TODO: emit error message on Err
-                self.entity_store
+                if let Err(e) = self
+                    .entity_store
                     .update(message.clone().try_into().unwrap())
```

Comment:
Need to handle this as well

Reply:
In this case, the message was previously checked to be a register message on line 113, so this unwrap never panics. That's not very obvious to see, so the API will be changed a bit to remove this unwrap in a later PR.